fs/fs.go

```diff
@@ -377,6 +384,30 @@ func (c *sociContext) populateImageLayerToSociMapping(sociIndex *soci.Index) {
 	}
 }
 
+// manifestMap is a wrapper for an atomic map
+// of a manifest digest to its appropriate list of diffIDs.
+//
+// key: manifest digest
+// val: map where
```

Comment:
This val map is only ever used to construct a `layerVerifier`, can we construct it eagerly instead?